File: Projects/2-Bouncer.py
age = input('How old are you? ')

if age != "":
    age = int(age)
    if age >= 21:
        print("You are good to enter and can drink!")
    elif age >= 18:
        print("You can enter, but need a wristband!")
    else:
        print("You can't come in little one! :(")
else:
    print("Please enter an age!")

File: Projects/2-Bouncer.js
const prompt = require('prompt-sync')()

let age = prompt("How old are you? ")

// When comparing a string with a number, js will convert the string to a number
if (age) {
  if (age >= 21) {
    console.log("You are good to enter and can drink!")
  }
  else if (age >= 18) {
    console.log("You can enter, but need a wristband!")
  }
  else {
    console.log("You can't come in little one! :(")
  }
}
else {
  console.log("Please enter an age!")
}